lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_get_udids.rb

```diff
@@ -0,0 +1,93 @@
+require 'fastlane/action'
+require 'open3'
+require 'shellwords'
+require 'googleauth'
+require_relative '../helper/firebase_app_distribution_helper'
+require_relative '../helper/firebase_app_distribution_error_message'
+require_relative '../client/firebase_app_distribution_api_client'
+require_relative '../helper/firebase_app_distribution_auth_client'
+
+module Fastlane
+  module Actions
+    class FirebaseAppDistributionGetUdidsAction < Action
+      extend Auth::FirebaseAppDistributionAuthClient
+      extend Helper::FirebaseAppDistributionHelper
+
+      def self.run(params)
+        auth_token = fetch_auth_token(params[:service_credentials_file], params[:firebase_cli_token])
+        fad_api_client = Client::FirebaseAppDistributionApiClient.new(auth_token, params[:debug])
+
+        app_id = params[:app]
+        udids = fad_api_client.get_udids(app_id)
+        write_udids_to_file(udids, params[:output_file])
```

Comment:
Consider printing a log statement if it's empty and not outputing to the file

Reply:
No seeing the change here. Did you maybe forget to push up your change?

Reply:
Oops, sorry, pushed it up.